src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,32 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
```

Comment:
```suggestion
interface Props = {
```
use interface prefer